site/_guides/cert-manager.md

```diff
@@ -517,14 +517,13 @@ $ curl https://httpbin.davecheney.com/get
 cert-manager currently does not have a way to interact with HTTPProxy objects in order to respond to the HTTP01 challenge correctly.
 (See [#950][10] and [#951][11] for details.)
 
-cert-manager does this by creating a new, temporary Ingress object that will direct requests from Let's Encrypt to temporary pods called 'solver pods'.
+cert-manager can be configured to request certificates automatically using an `Certificate` object.
+When cert-manager finds a `Certificate` object, it will implement the HTTP01 challenge by creating a new, temporary Ingress object that will direct requests from Let's Encrypt to temporary pods called 'solver pods'.
 These pods know how to respond to Let's Encrypt's challenge process for verifying you control the domain you're issuing certificates for.
-This means that cert-manager can't be *directly* used for generating certificates for HTTPProxy configuration.
 
-However, we can create a dummy Ingress object that will have cert-manager provision the certificate Secret, so that we can consume it in a HTTPProxy.
```

Comment:
I think it's useful to keep the part of this sentence that says that cert-manager will generate a TLS secret, which can be referred to by our HTTPProxy, just to close the loop on how this works for HTTPProxy.

Reply:
ok added back a sentence explaining this.